inc/namespace.php

```diff
@@ -74,6 +78,80 @@ function load_metadata() {
 	} );
 }
 
+/**
+ * Add filters to use Tachyon image URLs for metadata images.
+ * Image size and crop depend on the social media type.
+ */
+function use_tachyon_img_in_metadata() {
+	add_filter( 'hm.metatags.context.default', __NAMESPACE__ . '\\metadata_img_as_tachyon' );
+	add_filter( 'hm.metatags.context.twitter', __NAMESPACE__ . '\\metadata_img_as_tachyon_twitter' );
+	add_filter( 'hm.metatags.context.opengraph', __NAMESPACE__ . '\\metadata_img_as_tachyon_opengraph' );
+}
+
+/**
+ * Update twitter metadata to use Tachyon img URL.
+ *
+ * @param array $meta Twitter metadata.
+ *
+ * @return array Twitter metadata with image using Tachyon URL, if any.
+ */
+function metadata_img_as_tachyon_twitter( array $meta ) : array {
+	return metadata_img_as_tachyon( $meta, [
+		'width'  => 1200,
+		'height' => 600,
+		'crop'   => true,
+	] );
+}
+
+/**
+ * Update opengraph metadata to use Tachyon img URL.
+ *
+ * @param array $meta opengraph metadata.
+ *
+ * @return array opengraph metadata with image using Tachyon URL, if any.
+ */
+function metadata_img_as_tachyon_opengraph( array $meta ) : array {
+	return metadata_img_as_tachyon( $meta, [
+		'width'  => 1200,
+		'height' => 627,
+		'crop'   => false,
+	] );
+}
+
+/**
+ * Update metadata image URL to use Tachyon URL with specified image settings.
+ *
+ * @param array $meta          Metadata per social media type.
+ * @param array $img_settings Image settings: size and crop to be used in Tachyon URL.
+ *
+ * @return array Metadata with updated image URL using Tachyon, if an image is specified.
+ */
+function metadata_img_as_tachyon( array $meta, array $img_settings = [] ) : array {
+	// Stop - no image for metadata.
+	if ( ! $meta['image'] ) {
+		return $meta;
+	}
+
+	// Default image settings.
+	$img_settings = array_merge( [
```

Comment:
You'll need to do some massaging of this array to use it for tachyon, see here for the query arguments it takes https://www.altis-dxp.com/resources/docs/media/dynamic-images/#query-args-reference

So if it's a crop use `resize=<width>,<height>`, if it without a crop use `fit=<width>,<height>`

Reply:
You can remove this `array_merge` bit now as the query args are being passed from the other functions and you're setting a default above